### PR TITLE
Feature/lib js

### PR DESCRIPTION
**Pull request by kebetsi**

### CODE REVIEW

File: src/components/controller/ops/accept_access.js [hunk 1]
```diff
@@ -11,18 +11,19 @@ async function acceptAccess (ctx: Context): Promise<void> {
...
     ctx.clientData);
 
   // Notify register about accepted state
   const acceptedState: AcceptedAuthState = {
     status: ACCEPTED_STATUS,
-    username: ctx.user.username,
-    token: appAccess.token,
+    username: ctx.user.username, // to be deprecated
+    token: appAccess.token, // to be deprecated
+    apiEndpoint: ctx.pryv.pryvService.apiEndpointFor(ctx.user.username, appAccess.token),
```
Comment (kebetsi) [inline src/components/controller/ops/accept_access.js]: tu le reçois pas directement?

File: README.md [hunk 2]
```diff
@@ -6,17 +6,18 @@ These web pages are the "popup frame" that opens during the [app authorization p
...
 ## Prerequisites
 
-Node v8+, yarn v1+
+Node v8, yarn v1
 
 ## How to?
 
 | Task                                      | Command                        |
 | ----------------------------------------- | ------------------------------ |
+| Prepare dev environement                  | `yarn setup`                   |
```
Comment (kebetsi) [inline README.md]: environment

File: src/context.js [hunk 3]
```diff
@@ -11,14 +11,11 @@ type QueryParameters = {
...
 class Context {
-  domain: string;
   appId: string; // id of the web-auth app
-  requestingAppId: string; // id of the app requesting access
   language: string;
-  returnURL: ?string;
-  oauthState: ?string;
-  permissions: Permissions;
-  pollKey: string;
+  authState: AuthState; // used only in the context of a "Auth" process
```
Comment (perki) [inline src/context.js]: rename to accessState

File: src/context.js [hunk 4]
```diff
@@ -42,16 +44,33 @@ class Context {
   }
 
   async init () {
+    if (this.pollUrl) {
```
Comment (kebetsi) [inline src/context.js]: write method `function isEnteringFromAuthRequest() { return this.pollUrl != null; }`

**Pull request merged**